### ISSUE TITLE
Add AppCenter flatpak remote to etc/skel

### ISSUE THREAD

**Issue by danrabbit** [elementary/os#468]
According to @davidmhewitt:
> I think the solution to adding a user remote will be to add it to a working system and figure out what config file gets created in the home directory when that happens. It's a keyfile somewhere, I remember finding it once.
>
> Then we can just add that to /etc/skel in the image.

**Comment by cassidyjames** [comment]
It looks like `~/.local/share/flatpak/repo/config` has the following:

```ini
[remote "elementary"]
url=https://flatpak.elementary.io/elementary/
xa.title=elementary
gpg-verify=true
gpg-verify-summary=true
xa.comment=elementary OS first party applications
xa.description=elementary OS first party applications
xa.homepage=https://elementary.io/
```

**Comment by danrabbit** [comment]
@davidmhewitt @tintou Thinking about it, should we be shipping that file in `default-settings`?

**Comment by davidmhewitt** [comment]
Yeah, probably ought to ship it in a package, so it can be changed if necessary. The only reason why we have to do the other one in the build scripts is to get epiphany installed in the image. But there would probably be some clever way of doing that with a deb package too.
